src/components/mangaCard.tsx

```diff
@@ -0,0 +1,119 @@
+import Button from "@mui/material/Button";
+import Card from "@mui/material/Card";
+import CardActions from "@mui/material/CardActions";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import Grid from "@mui/material/Grid";
+import Typography from "@mui/material/Typography";
+import { useEffect, useRef, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import MangaApis from "../agent/api";
+import MangaList from "../models/mangaList";
+import appPaths from "../shared/paths";
+
+//coverArt: item.relationships.attributes.fileName,
+
+const T = () => {
```

Comment:
What is T what does it do?

Reply:
I still need to refactor, this will include renaming my components.